### ISSUE TITLE
I cannot "flutter run" my app on iOS with this lib

### ISSUE THREAD

**Issue by TamasBarta** [bcc-code/bccm-player#52]
Hi,

I'm not sure how to debug the issue in the title, so at this point I'm not sure if it is an issue with this library, or just my project setup (I'm not a native iOS developer, I only know native Android).

Here's the output of the console:

<details>

<summary>Console output</summary>

```
Resolving dependencies...
  _fe_analyzer_shared 61.0.0 (65.0.0 available)
  analyzer 5.13.0 (6.3.0 available)
  archive 3.4.4 (3.4.6 available)
  collection 1.17.2 (1.18.0 available)
  dart_style 2.3.2 (2.3.3 available)
  flutter_hooks 0.18.6 (0.20.3 available)
  flutter_lints 2.0.3 (3.0.0 available)
  flutter_native_splash 2.3.3 (2.3.4 available)
  flutter_plugin_android_lifecycle 2.0.16 (2.0.17 available)
  flutter_state_notifier 0.7.3 (1.0.0 available)
  flutter_svg 2.0.7 (2.0.8 available)
  freezed 2.4.1 (2.4.5 available)
  i18n_extension 6.0.0 (9.0.2 available)
  intl 0.17.0 (0.18.1 available)
  lints 2.1.1 (3.0.0 available)
  material_color_utilities 0.5.0 (0.8.0 available)
  meta 1.9.1 (1.11.0 available)
  package_info_plus 4.1.0 (4.2.0 available)
  path_provider_android 2.2.0 (2.2.1 available)
  petitparser 5.4.0 (6.0.1 available)
  platform 3.1.2 (3.1.3 available)
  riverpod 2.4.1 (2.4.4 available)
  stack_trace 1.11.0 (1.11.1 available)
  state_notifier 0.7.2+1 (1.0.0 available)
  stream_channel 2.1.1 (2.1.2 available)
  test_api 0.6.0 (0.6.1 available)
  vector_graphics 1.1.7 (1.1.9+1 available)
  vector_graphics_codec 1.1.7 (1.1.9+1 available)
  vector_graphics_compiler 1.1.7 (1.1.9+1 available)
  wakelock_plus 1.1.1 (1.1.3 available)
  web 0.1.4-beta (0.3.0 available)
  webview_flutter_wkwebview 3.9.1 (3.9.2 available)
  xml 6.3.0 (6.4.2 available)
Got dependencies!
Launching lib/main.dart on Az iPhone egy rakas szar in debug mode...
Automatically signing iOS for device deployment using specified development team in Xcode project: <redacted>
Running pod install...                                           2,648ms
Upgrading Pods-Runner.release.xcconfig
Upgrading Pods-Runner.profile.xcconfig
Upgrading Pods-Runner.debug.xcconfig
Running Xcode build...                                          
Xcode build done.                                           33.4s
Installing and launching...                                     
Could not run build/ios/iphoneos/Runner.app on 4df56d8a4d31dc84058107ce39736e5ddb98a2f3.
Try launching Xcode and selecting "Product > Run" to fix the problem:
  open ios/Runner.xcworkspace

Installing and launching...                                        27.2s
Error launching application on Az iPhone egy rakas szar.
```

</details>

When I open the project in Xcode, I see this:

<details>

<summary>Xcode stuff</summary>

<img width="872" alt="kép" src="https://github.com/bcc-code/bccm-player/assets/1753591/06f019d5-90c6-4db9-b119-0d64b8cd3dcd">

`@import bccm_player;` line yields: `Module 'bccm_player' not found`.

```
//
//  Generated file. Do not edit.
//

// clang-format off

#import "GeneratedPluginRegistrant.h"

#if __has_include(<bccm_player/BccmPlayerPlugin.h>)
#import <bccm_player/BccmPlayerPlugin.h>
#else
@import bccm_player;
#endif
```

</details>

Another important details could be that I can export archives and IPAs with no issues, and the exported app works well.

Thanks for the tips in advance.

**Comment by andreasgangso** [comment]
Sounds like xcode acting up, especially since you are able to build archives.

I'd start with `flutter clean`, `pod install` and all those things. You might need xcode 15.

**Comment by andreasgangso** [comment]
Xcode also has some bugs around those module errors, e.g. "module 'flutter' not found" is very normal, but it's only a visual bug, and build/run actually works.
So if you haven't tried clicking run, try that.

**Comment by TamasBarta** [comment]
Hi @andreasgangso,

thanks for having a look. I tried `futter clean`, and even `flutter clean && flutter pub get && cd ios && pod deintegrate && pod install && cd ..`, but that didn't help. Clicking run also fails with the same error. I made sure to make the minimum target 14.0 too everywhere, which seems to be realated in https://github.com/flutter/flutter/issues/53573

Since then I tracked down the issue to this line in my `Podfile`:

```
platform :ios, '14.0'
```

At some point this fixed something else for me, but now I had to revert, and nothing else seems to be broken so far 😅

Thanks for your help, and for this awesome library 🙂
